packages/@uppy/companion/src/server/jobs.js

```diff
@@ -54,3 +59,54 @@ const cleanUpFinishedUploads = (dirPath) => {
     })
   })
 }
+
+async function runPeriodicPing ({ urls, requestTimeout }) {
+  // Run requests in parallel
+  await Promise.all(urls.map(async (url) => {
+    try {
+      // TODO rewrite to use a non-deprecated request library
```

Comment:
i would like to post a key that is set via environment, then we can hand out such random keys and recognize customers by them